addons/website/static/src/builder/plugins/options/social_media_option_plugin.js

```diff
@@ -328,14 +317,15 @@ class SocialMediaOptionPlugin extends Plugin {
      */
     removeIconClasses(el) {
         const iconEl = el.querySelector(ICON_SELECTOR);
-        if (iconEl) {
-            // Remove every fa classes except fa-x sizes.
-            for (const c of iconEl.classList) {
-                if (/^fa-[^0-9]/.test(c)) {
-                    iconEl.classList.remove(c);
-                }
-            }
+        if (!iconEl) {
+            return;
         }
+        // Remove every fa classes except fa-x sizes.
+        Array.from(iconEl.classList).forEach((c) => {
+            if (/^fa-[^0-9]/.test(c)) {
+                iconEl.classList.remove(c);
+            }
+        });
```

Comment:
isn't whats removed and whats added the same ? And performance wise the new one is not good..